ui/webui/src/app.jsx

```diff
@@ -33,15 +41,60 @@ export const Application = () => {
 
     useEffect(() => cockpit.file("/run/anaconda/bus.address").watch(setAddress), []);
 
+    const steps = [
+        {
+            id: 0,
+            name: "Installation language",
+            component: <AddressContext.Provider value={address}>
+                <InstallationLanguage />
+            </AddressContext.Provider>
+        },
+        {
+            id: 1,
+            name: "Storage configuration",
+            component: <AddressContext.Provider value={address}>
+                <InstallationDestination />
+            </AddressContext.Provider>,
+        },
+        {
+            id: 2,
+            name: "Review",
+            component: <AddressContext.Provider value={address}>
+                <ReviewConfiguration />
+            </AddressContext.Provider>,
+            nextButtonText: "Begin installation"
+        },
+        {
+            id: 3,
+            name: "Installation progress",
+            component: <AddressContext.Provider value={address}>
+                <InstallationProgress />
+            </AddressContext.Provider>,
+            nextButtonText: "Next"
+        },
+        {
+            id: 4,
+            name: "Installation complete",
+            component: <AddressContext.Provider value={address}>
+                <InstallationComplete />
+            </AddressContext.Provider>,
+            nextButtonText: "Reboot"
+        }
+    ];
+    const title = "Anaconda Installer";
+
     return (
         <Page>
-            <AddressContext.Provider value={address}>
-                {!path.length > 0 && <InstallationLanguage />}
-            </AddressContext.Provider>
-            {path.length > 0 &&
-            <AddressContext.Provider value={address}>
-                <Summary />
-            </AddressContext.Provider>}
+            <Wizard
+              navAriaLabel={`${title} steps`}
+              mainAriaLabel={`${title} content`}
+              titleId="wiz-modal-demo-title"
+              descriptionId="wiz-modal-demo-description"
+              title="Fedora Rawhide installation"
+              description="PRE-RELEASE/TESTING"
```

Comment:
Patternfly does not translate strings by itself. For this reason they expose through their properties the labels, so you can overwrite the english text.
For example here you need: 
``` 
nextButtonText={_("Next")}
backButtonText... 

etc
```

Find here https://www.patternfly.org/v4/components/wizard/#wizard all string properties that are labels, aria-labels or button texts.

Reply:
Adding there.